<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>768</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Archivo SemiBold SemiBold</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea_address">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>474</width>
      <height>769</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(148, 145, 226, 130), stop:1 rgba(175, 242, 216, 130));</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="0">
       <red>214</red>
       <green>221</green>
       <blue>218</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QMdiArea" name="mdiArea_pizza">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>611</width>
      <height>769</height>
     </rect>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QLabel" name="l_customerInfo">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>24</y>
      <width>280</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Customer Info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>80</y>
      <width>402</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Archivo SemiBold SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid black;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>675</y>
      <width>90</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_customerInfo_2">
    <property name="geometry">
     <rect>
      <x>607</x>
      <y>24</y>
      <width>339</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pizza Order</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lastNameLabel">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>140</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastNameLineEdit">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>172</y>
      <width>192</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="streetLineEdit">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>282</y>
      <width>412</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="streetLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>250</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*Street</string>
    </property>
   </widget>
   <widget class="QLabel" name="aptNoLabel">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>363</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apt No.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>392</y>
      <width>241</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityLabel">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>360</y>
      <width>54</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="postalCodeLineEdit">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>502</y>
      <width>192</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="postalCodeLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>470</y>
      <width>148</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*Postal Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstNameLineEdit">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>172</y>
      <width>192</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstNameLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>140</y>
      <width>137</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*First Name</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>80</y>
      <width>486</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Archivo SemiBold SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid black;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>91</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pizza size</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>487</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.714286, y1:0.25, x2:0, y2:1, stop:0 rgba(148, 145, 226, 130), stop:1 rgba(175, 242, 216, 130));
border-radius: 10px;</string>
    </property>
    <widget class="QRadioButton" name="small_radioButton">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>60</y>
       <width>61</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
    background-color: transparent; /* Set the background color of the entire radio button */
    color: transparent; /* Set the text color */
}

QRadioButton::indicator {
    width: 35px; /* Set the width of the indicator (the circle) */
    height: 35px; /* Set the height of the indicator */
    border: 10px solid #ffffff; /* Set the border of the indicator */
    border-radius: 27px; /* Make the indicator a circle by setting border-radius */
    background-color: #fff; /* Set the background color of the indicator */
}

QRadioButton::indicator:checked {
    background-color: qlineargradient(spread:pad, x1:0.714286, y1:0.25, x2:0, y2:1, stop:0 rgba(148, 145, 226, 150), stop:1 rgba(175, 242, 216, 150));
}
</string>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="large_radioButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>101</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
    background-color: transparent; /* Set the background color of the entire radio button */
    color: transparent; /* Set the text color */
}

QRadioButton::indicator {
    width: 80px; /* Set the width of the indicator (the circle) */
    height: 80px; /* Set the height of the indicator */
    border: 10px solid #ffffff; /* Set the border of the indicator */
    border-radius: 50px; /* Make the indicator a circle by setting border-radius */
    background-color: #fff; /* Set the background color of the indicator */
}

QRadioButton::indicator:checked {
    background-color: qlineargradient(spread:pad, x1:0.714286, y1:0.25, x2:0, y2:1, stop:0 rgba(148, 145, 226, 150), stop:1 rgba(175, 242, 216, 150));
}

</string>
     </property>
     <property name="text">
      <string>Large</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="medium_radioButton">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>40</y>
       <width>81</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
    background-color: transparent; /* Set the background color of the entire radio button */
    color: transparent; /* Set the text color */
}

QRadioButton::indicator {
    width: 60px; /* Set the width of the indicator (the circle) */
    height: 60px; /* Set the height of the indicator */
    border: 10px solid #ffffff; /* Set the border of the indicator */
    border-radius: 40px; /* Make the indicator a circle by setting border-radius */
    background-color: #fff; /* Set the background color of the indicator */
}

QRadioButton::indicator:checked {
    background-color: qlineargradient(spread:pad, x1:0.714286, y1:0.25, x2:0, y2:1, stop:0 rgba(148, 145, 226, 150), stop:1 rgba(175, 242, 216, 150));
}

</string>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>159</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: black;</string>
     </property>
     <property name="text">
      <string>Large</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>140</y>
       <width>159</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: black;</string>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>159</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: black;</string>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>487</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.514286, y1:1, x2:0, y2:1, stop:0 rgba(148, 145, 226, 130), stop:1 rgba(175, 242, 216, 130));
border-radius: 10px;</string>
    </property>
    <widget class="QCheckBox" name="checkBox_onion">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>92</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Onion</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_olive">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Black Olives</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_greenPepper">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>54</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Green Pepper</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ham">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>16</y>
       <width>232</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Ham</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mushroom">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>92</y>
       <width>232</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Mushroom</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cheese">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>168</y>
       <width>232</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Extra Cheese</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pepper">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>54</y>
       <width>232</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Pepper</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_artichokes">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>130</y>
       <width>232</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Artichokes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pineapple">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>16</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Armata</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    background-color: transparent; /* Set the background color of the entire checkbox */
    color: black; /* Set the text color */
}

QCheckBox::indicator {
    width: 15px; /* Set the width of the indicator (the checkmark) */
    height: 15px; /* Set the height of the indicator */
    
}

</string>
     </property>
     <property name="text">
      <string>Pineapple</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>321</y>
      <width>381</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Toppings (Max 3 toppings)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>580</y>
      <width>90</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_order">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>675</y>
      <width>90</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>392</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color:#ffffff;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Armata</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>* Please complete all required fields.</string>
    </property>
   </widget>
   <zorder>mdiArea_pizza</zorder>
   <zorder>mdiArea_address</zorder>
   <zorder>l_customerInfo</zorder>
   <zorder>line</zorder>
   <zorder>pushButton_cancel</zorder>
   <zorder>l_customerInfo_2</zorder>
   <zorder>lastNameLabel</zorder>
   <zorder>lastNameLineEdit</zorder>
   <zorder>streetLineEdit</zorder>
   <zorder>streetLabel</zorder>
   <zorder>aptNoLabel</zorder>
   <zorder>cityLineEdit</zorder>
   <zorder>cityLabel</zorder>
   <zorder>postalCodeLineEdit</zorder>
   <zorder>postalCodeLabel</zorder>
   <zorder>firstNameLineEdit</zorder>
   <zorder>firstNameLabel</zorder>
   <zorder>line_2</zorder>
   <zorder>label</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_reset</zorder>
   <zorder>pushButton_order</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>postalCodeLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
